sdk/resourcemanager/azure-resourcemanager-resources/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 2.30.0-beta.1 (2023-08-11)
```

Comment:
The date is tomorrow?

Reply:
Yeah, they say they've customer waiting for it.

core/identity release should be almost done, anyway beta release from release branch should not affect or be affected by that.